src/components/calculator-component.js

```diff
@@ -0,0 +1,11 @@
+function calculator(bodyId,screenId,inputId,buttonsGroup) {
+    return`
+    <div id="${bodyId}">
+        <div id="${screenId}">
+            <input type="text" placeholder="0" id="${inputId}">
+        </div>
+            ${buttonsGroup}
```

Comment:
it must be in the same level of div!
then back one tab!

Reply:
no no.it's in the same level, it's because the buttons-group component already have an tab at the beginning

Reply:
But looking for this this identation it's wrong!
IMO you should keep this `${buttonsGroup}` in the same level of `div` above ☝🏼